credit_integration/models.py

```diff
@@ -0,0 +1,87 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from enumfields import EnumField
+
+from credit_integration.enums import CreditDecisionStatus
+from leasing.models import Contact
+from leasing.models.mixins import NameModel, TimeStampedModel
+from users.models import User
+
+
+class CreditDecisionReason(TimeStampedModel):
+    """
+    In Finnish: Luottopäätöksen syyn perusteet
+    """
+
+    # In Finnish: Syykoodi
+    reason_code = models.CharField(
+        verbose_name=_("Reason code"), max_length=3, unique=True,
+    )
+
+    # In Finnish: Syy
+    reason = models.TextField(verbose_name=_("Reason"),)
+
+
+class CreditDecision(TimeStampedModel):
+    """
+    In Finnish: Luottopäätös
+    """
+
+    # In Finnish: Asiakas
+    customer = models.ForeignKey(
+        Contact,
+        verbose_name=_("Customer"),
+        related_name="credit_decisions",
+        blank=True,
+        null=True,
+        on_delete=models.CASCADE,
+    )
+
+    # In Finnish: Luottopäätöksen tila
+    status = EnumField(CreditDecisionStatus, verbose_name=_("Status"), max_length=30)
+
+    # In Finnish: Luottopäätöksen perusteet
+    reasons = models.ManyToManyField(CreditDecisionReason, verbose_name=_("Reasons"))
+
+    # In Finnish: Y-tunnus
+    business_id = models.CharField(
+        verbose_name=_("Business ID"), null=True, blank=True, max_length=9,
```

Comment:
Using `null` for "non-empty" values on string based fields should be avoided (https://docs.djangoproject.com/en/3.2/ref/models/fields/#null). Same with the other `CharFields` in this file.

Reply:
These are fixed now.